packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -383,8 +379,8 @@ export class GcdsSelect {
             {...attrsSelect}
             id={selectId}
             onBlur={() => this.onBlur()}
-            onFocus={() => this.onFocus()}
-            onChange={e => this.handleChange(e)}
+            onFocus={() => this.gcdsFocus.emit()}
+            onInput={e => this.handleChange(e)}
```

Comment:
This is super interesting. As we're doing an `onInput`, does it matter, or not, if we only have an external API for change `gcdsChange`? Will it make sense to be consistent and either also have `gcdsInput` or change `gcdsChange` to `gcdsInput`?

`onChange` occurs when the element loses focus and its value has changed
`onInput` occurs immediately when the element receives an input, or when its value has changed